easybuild/easyconfigs/r/Ruby-bundle/Ruby-bundle-2.3.4-Ruby-2.3.4.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
+
+name = 'Ruby-bundle'
+version = '2.3.4'
+versionsuffix = '-Ruby-%s' % version
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
```

Comment:
Please add some notes about the purpose of this bundle, beyond the main `Ruby` easyconfig.

Reply:
@shahzebsiddiqui Sorry for only picking up on this now, it suddenly bumped into view...

Any reason for not adding these additional extension to the `Ruby` easyconfig instead?

Reply:
@boegel oh this was an old easyconfig we had, I was not sure the best way to handle this since I wanted a base ruby install and add ruby package in separate easyconfig so that there is not one monolithic Ruby install with many ruby packages. Since there was no concept of ``Ruby-bundle`` I had to create something and found there was no easyconfig so I decided to create a PR.  If you don't think this is useful then please disregard this.

Reply:
Thanks for the info. Given the age of this PR, I doubt it's still useful to pursue this, so I'll close it.